src/main/cli/cli.c

```diff
@@ -4956,7 +4956,7 @@ static void cliRcSmoothing(const char *cmdName, char *cmdline)
     UNUSED(cmdline);
     rcSmoothingFilter_t *rcSmoothingData = getRcSmoothingData();
     cliPrint("# RC Smoothing Type: ");
-    if (rxConfig()->rc_smoothing_type == RC_SMOOTHING_TYPE_FILTER) {
+    if (rxConfig()->rc_smoothing_mode == ON) {
```

Comment:
This should be treated as a boolean. Re-defining enums for boolean is an antipattern.

Reply:
OK, I left the data type as `uint8_t` so we didn't have to change msp, but it is treated as a 0 or 1 boolean now.

Reply:
Yes, this is the correct approach. All parameters in parameter groups should be using types with an explicit storage length, to ensure the data structure is well defined.

Reply:
Great, thanks.